commands/cert/autocertificate/create.go

```diff
@@ -0,0 +1,97 @@
+package autocertificate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ionos-cloud/ionosctl/v6/commands/cert/provider"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/json2table/jsonpaths"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/tabheaders"
+	"github.com/ionos-cloud/sdk-go-bundle/products/cert/v2"
+	"github.com/spf13/viper"
+)
+
+func AutocertificatePostCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "certmanager",
+		Resource:  "autocertificate",
+		Verb:      "create",
+		Aliases:   []string{"post", "c"},
+		ShortDesc: "Create an AutoCertificate. Create a zone with the same name as the common-name parameter and confirm that the zone is enabled",
```

Comment:
Nice, the extra zone text is useful, but let's reword to let users know what do we mean by 'zone' 

```suggestion
		ShortDesc: "Create an AutoCertificate. Requires an enabled DNS Zone with the same name as the --common-name.",
```